frontend/src/charts/choroplethMap/RateMapLegend.tsx

```diff
@@ -263,14 +151,40 @@ export default function RateMapLegend(props: RateMapLegendProps) {
 
   return (
     <section
-      className='mx-4 flex w-full flex-col items-center text-left'
+      className={`mx-4 flex w-full flex-col items-start text-left ${
+        props.isMulti ? 'md:mx-auto md:w-1/2' : ''
+      }`}
+      aria-label='Legend for rate map'
       ref={containerRef}
     >
-      <ClickableLegendHeader
-        legendTitle={props.legendTitle}
-        dataTypeConfig={props.dataTypeConfig}
-      />
-      <svg ref={svgRef} className='w-full' />
+      <div className='w-full'>
+        <div className='flex flex-col items-center'>
+          <ClickableLegendHeader
+            legendTitle={props.legendTitle}
+            dataTypeConfig={props.dataTypeConfig}
+          />
+
+          <div
+            // common classes across all views
+            className={`w-2/3 columns-1 tiny:columns-2 gap-1 space-y-1 border-0 border-greyGridColorDarker border-t-[1px] border-solid px-4 pt-4 ${
+              props.isMulti
+                ? // multimap only
+                  'columns-auto sm:columns-3 lg:columns-4'
+                : props.isCompareMode
+                  ? // compare mode only
+                    'smMd:columns-3 md:columns-2 lg:columns-3'
+                  : // non-compare mode only
+                    'sm:columns-1'
+            }`}
```

Comment:
I don't love this nested ternary class name stuff, but it was the only way i could get it working and still maintain vscode recognizing tailing classes with autocomplete / linting etc. and that seemed worth keeping. I tried to nicely comment to explain the logic flow